vidarr-cromwell/src/main/java/ca/on/oicr/gsi/vidarr/cromwell/CromwellOutputProvisioner.java

```diff
@@ -88,11 +88,11 @@ public String type() {
           new Pair<>(".gtf", "text/gtf"),
           new Pair<>(".html", "text/html"),
           new Pair<>(".vcf", "text/vcf"),
-          new Pair<>(".txt.gz", "application/txt-gz"),
-          new Pair<>(".gz", "application/txt-gz"),
-          new Pair<>(".out", "txt/plain"),
-          new Pair<>(".log", "txt/plain"),
-          new Pair<>(".txt", "txt/plain"),
+          new Pair<>(".txt.gz", "application/text-gz"),
+          new Pair<>(".gz", "application/text-gz"),
+          new Pair<>(".out", "text/plain"),
+          new Pair<>(".log", "text/plain"),
+          new Pair<>(".txt", "text/plain"),
           new Pair<>(".junction", "txt/junction"),
```

Comment:
Should this one be changed to "text" too?

Reply:
Yes. Thank you.